testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -500,11 +497,21 @@ def test_fasta2select_ClustalW(self, tmpdir):
         # numbers computed from alignment with clustalw 2.1 on Mac OS X
         # [orbeckst] length of the output strings, not residues or anything
         # real...
-        assert len(sel['reference']) == 23080, ("selection string has"
-                                                "unexpected length")
-        assert len(
-            sel['mobile']) == 23090, "selection string has unexpected length"
-
+        assert len(sel['reference']) == 23080, self.error_msg
+        assert len(sel['mobile']) == 23090, self.error_msg
+
+    def test_fasta2select_nogap(self, tmpdir):
+        """test align.fasta2select() on aligned FASTA with no gap in resid
+        (Issue #3124)"""
+        ref_resids = [x for x in range(705)]
+        target_resids = [x for x in range(705)]
```

Comment:
It's not a lot, but this is an unnecessary operation, rather than having two separate lists, couldn't you just assign one since they are both identical?